1-Draft/RFCXXXX-Policy.md

```diff
@@ -0,0 +1,236 @@
+|                                  | ModuleLogging               | ModuleNames                        | String | Computer, Then User |
+|                                  | ProtectedEventLogging       | EncryptionCertificate              | DWORD  | Computer Wide       |
+|                                  | ScriptBlockLogging          | EnableScriptBlockInvocationLogging | DWORD  | Computer, Then User |
+|                                  | ScriptBlockLogging          | EnableScriptBlockLogging           | DWORD  | Computer, Then User |
+|                                  | Transcription               | EnableTranscripting                | DWORD  | Computer, Then User |
+|                                  | Transcription               | EnableInvocationHeader             | DWORD  | Computer, Then User |
+|                                  | Transcription               | OutputDirectory                    | String | Computer, Then User |
+|                                  | UpdatableHelp               | DefaultSourcePath                  | String | Computer Wide       |
+
+#### JSON file settings format
+
+```json
+{
+  "PowerShell": {
+    "RegularSettings": {
+      "ConsoleSessionConfiguration": {
+        "EnableConsoleSessionConfiguration": true,
+        "ConsoleSessionConfigurationName": "name"
+      },
+      "ProtectedEventLogging": {
+        "EnableProtectedEventLogging": false,
+        "EncryptionCertificate": [
+          "Joe"
+        ]
+      },
+      "ScriptBlockLogging": {
+        "EnableScriptBlockInvocationLogging": true,
+        "EnableScriptBlockLogging": false
+      },
+      "ScriptExecution": {
+        "ExecutionPolicy": "RemoteSigned",
+        "PipelineMaxStackSizeMB": 10
+      },
+      "Transcription": {
+        "EnableTranscripting": true,
+        "EnableInvocationHeader": true,
+        "OutputDirectory": "c:\\tmp"
+      },
+      "UpdatableHelp": {
+        "DefaultSourcePath": "f:\\temp"
+      }
+    },
+
+    "PoliciesSettings": {
+      ...
+    }
+  },
+
+  "OtherPowerShellApplication": {
+    "RegularSettings": {
+      ...
+    },
+    "PolicySettings": {
+      ...
+    }
+}
+```
+
+## Alternate Proposals and Considerations
+
+### Automatically resolve Windows PowerShell policy conflicts
+
+We could attempt to resolve policy conflicts between PowerShell 7 policy and Windows PowerShell policy.
+This would make the `Precedence for Policy settings` not just a simple list but a complex set of rules that would not be easily understood.  See [this conversation](https://github.com/PowerShell/PowerShell/issues/9309?#issuecomment-480643922).
+
+### Allow admins to overwrite computer-wide settings
+
+In System Lock-down mode, we attempt to protect from the admin,
+so allowing computer-wide or policy setting to be overwritten at the command-line is dangerous.
+
+We could try to check for System Lock-down mode and
+admin user and allow `-settingsfile` to overwrite computer-wide settings.
```

Comment:
We should not allow `-SettingsFile` to be processed if in system lock down.  The perf impact may be ok since it's only when this is specified.